--- /etc/minidlna.conf	2019-06-10 19:08:29.570684928 +0000
+++ minidlna.conf.old	2019-06-10 19:07:46.877654790 +0000
@@ -23,17 +23,17 @@
 #   * "P" for pictures (eg. media_dir=P,/var/lib/minidlna/pictures)
 #   * "V" for video    (eg. media_dir=V,/var/lib/minidlna/videos)
 #   * "PV" for pictures and video (eg. media_dir=PV,/var/lib/minidlna/digital_camera)
-media_dir=/var/lib/minidlna
+media_dir=/media/dysk1/minidlna
 
 # Set this to merge all media_dir base contents into the root container
 # (The default is no.)
 #merge_media_dirs=no
 
 # Path to the directory that should hold the database and album art cache.
-db_dir=/var/cache/minidlna/db
+db_dir=/media/dysk1/minidlna/db
 
 # Path to the directory that should hold the log file.
-log_dir=/var/log
+log_dir=/media/dysk1/minidlna/log
 
 # Type and minimum level of importance of messages to be logged.
 #
@@ -78,7 +78,7 @@
 
 # Name that the DLNA server presents to clients.
 # Defaults to "hostname: username".
-#friendly_name
+friendly_name=Freya
 
 # Serial number the server reports to clients.
 # Defaults to the MAC address of nework interface.
